### PR TITLE
change to feature

### PR DESCRIPTION
**Pull request by cjsim89**

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -1 +1 @@
-and now there is text here. here is a test in my file . more text.
+here is a test in my file and here is a change
```
Comment (AliciaWatt) [inline test.txt:1]: great change!

**Review by AliciaWatt — APPROVED** [review]
looks great

**Comment by AliciaWatt** [comment]
I left a few comments. looks amaze

File: test.txt (hunk 1 above)
Comment (cjsim89) [reply test.txt:1]: Thanks I worked really hard on this file.

**Pull request merged**